tests/test_utils.py

```diff
@@ -120,3 +122,11 @@ def test_multilogit_inv(test_input, test_output):
     res = multilogit_inv(tt.as_tensor_variable(test_input))
     res = res.eval()
     assert np.array_equal(res.round(2), test_output)
+
+
+def test_compute_steady_state():
+
+    P = tt.as_tensor_variable(np.arange(4).reshape(2, 2).astype(np.float32))
+
+    ss_probs = compute_steady_state(P)
+    assert isinstance(ss_probs, TensorVariable)
```

Comment:
This isn't the way we want to be writing this test, but I'm a little stuck on coming up with an alternate approach. ss_probs looks something like this:

```python
In [6]: ss_probs
Out[6]: Solve{A_structure='general', lower=False, overwrite_A=False, overwrite_b=False}.0
```

and it has a lot of things that can be called. It seemed like maybe we could assert that the probabilities sum to 1 with something like

```python
ss_probs.sum()
Sum{acc_dtype=float64}.0
```

but the output of that doesn't seem to be a float, it's a TensorVariable itself

Reply:
You're dealing with symbolic variables in Aesara/Theano here.  When you want to compute values for the those symbolic operations, you need to compile their symbolic graphs using `theano.function`.  See the documentation [here](https://aesara.readthedocs.io/en/latest/introduction.html#sneak-peek).